src/util/sync/mod.rs

```diff
@@ -29,7 +29,7 @@ use tokio::{
     time::{self, Duration, Interval},
 };
 
-use self::translation::{import_sync_records, SyncRecord, SyncType, TRANSLATION_RECORDS};
+use self::translation::{import_sync_records, TRANSLATION_RECORDS};
```

Comment:
```suggestion
use translation::{import_sync_records, TRANSLATION_RECORDS};
```

`translation` module already declared above so should be OK?

Reply:
thanks, I removed it